src/pages/wishlist-page/WishlistPage.jsx

```diff
@@ -0,0 +1,64 @@
+import { Navigation } from "../../components";
+import "./wishlist-page.css";
+import { Link } from "react-router-dom";
+import { useProduct } from "../../context";
+
+export default function Wishlist() {
+  const {
+    ProductState: { wishlist, cart },
+    ProductDispatch,
+  } = useProduct();
+  return (
+    <div>
+      <Navigation />
+      <h1 class="wishlist-heading center">My Wishlist</h1>
+      <div className="wishlist-flex">
+        {wishlist.map((w) => {
+          return (
+            <div className="wishlist-container">
+              <div className="wishlist-card">
+                <img className="card-image" src={w.image} alt={w.alt}></img>
+                <div class="description">
+                  <p class="card-heading center">{w.name}</p>
+                  <p className="product-description">{w.description}</p>
+                  <div class="flex product-price">
+                    <p>Rs. {w.price}</p>
+                    <p className="original-price">Rs. {w.originalPrice} </p>
+                    <p>{w.discount}% off</p>
+                  </div>
+                  <div className="flex product-button">
+                    {cart.some((c) => c.id === w.id) ? (
+                      <Link to="/Cart">
+                        <button class="ecom-btn">Move To Cart</button>
+                      </Link>
```

Comment:
Suggestion : w doesnt make any sense , if you are using general names , name it item